src/main/resources/org/jenkinsci/plugins/ghprb/upstream/GhprbUpstreamStatus/config.jelly

```diff
@@ -5,13 +5,10 @@
   <f:entry title="${%Commit Status URL}" field="statusUrl"> 
     <f:textbox default="${descriptor.getStatusUrlDefault(instance)}" placeholder="${descriptor.getStatusUrlDefault(instance)}"/>
   </f:entry>  
-  <f:entry title="${%Commit Status Build Triggered}" field="triggeredStatus">
```

Comment:
This field serves no purpose in the case of UpstreamStatus since onTriggered() is never (and could not) be fired, so it gives a false sense of functionality to users. The config.jelly for the normal SimpleStatus in the trigger section still has it.